scripts/canary-release.js

```diff
@@ -0,0 +1,99 @@
+/* eslint-disable */
+const semver = require('semver');
+const cp = require('child_process');
+const { basename } = require('path');
+
+const { read: readConfig } = require('@changesets/config');
+const readChangesets = require('@changesets/read').default;
+const assembleReleasePlan = require('@changesets/assemble-release-plan')
+  .default;
+const applyReleasePlan = require('@changesets/apply-release-plan').default;
+const { getPackages } = require('@manypkg/get-packages');
+
+function getNewVersion(version, type) {
+  const gitHash = cp
+    .spawnSync('git', ['rev-parse', '--short', 'HEAD'])
+    .stdout.toString()
+    .trim();
+
+  return semver.inc(version, `pre${type}`, true, 'alpha-' + gitHash);
```

Comment:
this should say canary if it's going to be on each PR merge. don't want to create confusion

Reply:
Do you mean to change to `canary` tag instead of `alpha` tag?

Reply:
yeah, what do you think of that?

Reply:
Sure! I'll change that :)